script/results.html

```diff
@@ -62,6 +62,11 @@
   <button class="w3-button w3-right w3-light-grey" id="download" type="submit" onclick=download()
     hidden="hidden">Download Results Feedback</button>
 
+    <button class="w3-button w3-right w3-light-grey" id="downloadjson" type="submit" onclick=downloadjson()
```

Comment:
result.html updated, regenerate the result-embed.html (same script used to regenerate catalog)

Reply:
i did rebase

Reply:
i did rebase